src/components/Training/Modal.js

```diff
@@ -79,8 +79,18 @@ const Close = styled.button`
   `}
 `
 
+const ModalStyles = createGlobalStyle`
```

Comment:
Cleaner way pf setting global styles

Reply:
this rules target the body? nice

Reply:
Global styles now allow for props! Its great!